### ISSUE TITLE
Investigate MongoDB upgrade

### ISSUE THREAD

**Issue by j987987** [unfetter-discover/unfetter#1357]
MongoDB is up to version 4.1.X, when we're on version 3.4.  Previous attempts at updating failed due to breaking some of the aggregation pipelines, but it's worth looking into again.

**Comment by j987987** [comment]
I brought up the stack using MongoDB 4.1 and reviewed the dashboards with @mpmmasterpeace , and it appears to work.  The more complicated issue would be handling the upgrade of the raw data.

**Comment by j987987** [comment]
I follow the instructions to upgrade 3.4 => 3.6 then 3.6 => 4.0.  I'm not sure if the upgrade step to 3.6 is necessary, but I didn't get some of the messages I did when I tried to go straight from 3.4 to 4.1 (keep in mind, 4.1 isn't a "stable" release so 4.0 may be safer.)

https://docs.mongodb.com/manual/release-notes/3.6-upgrade-standalone/
https://docs.mongodb.com/manual/release-notes/4.0-upgrade-standalone/

1. Bring up DB with mongo:3.6, run `db.adminCommand( { setFeatureCompatibilityVersion: "3.6" } )`
2. Bring up DB with mongo:4.0, run `db.adminCommand( { setFeatureCompatibilityVersion: "4.0" } )`
